style.css

```diff
@@ -0,0 +1,16 @@
+body {
+  background-color: darkolivegreen;
+}
+
+h1 {
+  text-align: center;
+  color: aliceblue;
+}
+
+/* .section {
+  width: 200px;
+  height: auto;
+  padding: auto;
+  margin: 2rem;
+
+} */
```

Comment:
Please remove this and any commented code to maintain readability in your project.